openmmtools/integrators.py

```diff
@@ -1520,7 +1520,13 @@ def reset_integrator(self):
 
 
 class ExternalPerturbationLangevinIntegrator(LangevinIntegrator):
-    """LangevinSplittingIntegrator that accounts for external perturbations and tracks protocol work."""
+    """
+    LangevinSplittingIntegrator that accounts for external perturbations and tracks protocol work.
+
+    If used before an NCMC attempt, initialize the protocol work with
```

Comment:
This description is a bit vague. Why not write something along the lines of

`Reinitialize protocol work by using...` and leave NCMC out of it. 

This might also be a good opportunity to add the following to the docstring, right before this segment.

`The externally performed protocol work is accumulated in the protocol_work global variable.`